test/e2e/mock-e2e.js

```diff
@@ -4,6 +4,9 @@ const blacklistedHosts = [
   'mainnet.infura.io',
   'rinkeby.infura.io',
   'ropsten.infura.io',
+  'arbitrum-mainnet.infura.io',
```

Comment:
nit: could we keep the alphabetic order of the array?

Reply:
Done.